### PR TITLE
Fix for issue#10

### PR DESCRIPTION
**Pull request by tahir24434**
https://github.com/networkop/meshnet-cni/issues/10
daemon/generated in proto_base is at path /go/daemon/generated. In build_base, we are modifying the Workdir.
Without this change, we hit below error 

`failed to solve with frontend dockerfile.v0: failed to build LLB: failed to compute cache key: failed to walk /var/lib/docker/tmp/buildkit-mount009672704/daemon: lstat /var/lib/docker/tmp/buildkit-mount009672704/daemon: no such file or directory`

### CODE REVIEW

**Comment by networkop** [comment]
so does it build successfully with this change?

**Comment by tahir24434** [comment]
Yes. with this change, it builds fine.

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,50 +1,37 @@
...
 RUN go mod download
 
-FROM build_base AS build
 
+FROM build_base AS build
 COPY . .
-
-COPY --from=proto_base daemon/generated daemon/generated
-RUN CGO_ENABLED=0 GOOS=linux go build -o meshnet plugin/meshnet.go 
+COPY --from=proto_base /go/daemon/generated daemon/generated
```
Comment (tahir24434) [inline docker/Dockerfile]: This is the only real change in this commit.  
- COPY --from=proto_base daemon/generated daemon/generated
+ COPY --from=proto_base /go/daemon/generated daemon/generated

Rest of the changes are just removing the extra lines within each build stage.

Comment (networkop) [reply docker/Dockerfile]: yep, LGTM. I just want to test it locally before merging. will do this in the next couple of days.

Comment (tahir24434) [reply docker/Dockerfile]: Sure. Thanks.

**Pull request merged**